source/lambda/customResourceKendraDataSource/lambda_function.py

```diff
@@ -23,11 +23,11 @@ def on_create(event, context):
     
     create_faq_response = kendra_client.create_faq(
         IndexId=kendraIndexId,
-        Name="DUSCovidFAQ",
-        Description='covid-19 questions and answers',
+        Name="Medical FAQ",
```

Comment:
You cannot have a blank space in the FAQ name:
FAQ name needs to satisfy the regular  expression pattern: [a-zA-Z0-9][a-zA-Z0-9_-]* 
--

Reply:
I was aiming for this PR to be focused on content updates for new pdfs and faq.  But I can bundled covid name changes in code, this will take more time and require install testing though.